dynawo/sources/Common/DYNClone.hpp

```diff
@@ -0,0 +1,44 @@
+//
+// Copyright (c) 2021, RTE (http://www.rte-france.com)
+// See AUTHORS.txt
+// All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at http://mozilla.org/MPL/2.0/.
+// SPDX-License-Identifier: MPL-2.0
+//
+// This file is part of Dynawo, an hybrid C++/Modelica open source time domain
+// simulation tool for power systems.
```

Comment:
the new line of the copyright is "This file is part of Dynawo, an hybrid C++/Modelica open source suite of simulation tools for power systems."

Reply:
OK